Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>node_modules/\r\n 
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 5f81f3c73b865cc502cedc90110b8a7c8cff64fd)
+++ b/.gitignore	(date 1773319369695)
@@ -1,2 +1,2 @@
 node_modules/
- 
\ No newline at end of file
+.env
\ No newline at end of file
